contracts/satoshi-bridge/src/zcash_utils/psbt_wrapper.rs

```diff
@@ -61,12 +64,28 @@ impl PsbtWrapper {
             vin.len()
         ];
 
+        // TODO: pass the recipient address and amount to verify the orchard output
+        // let recipient_address = "<SOME ZCASH ADDRESS>";
+        // let value = "<Amount of the output>";
+
+        // TODO: verify orchard bundle
+        // How to verify orchard bundle value and recipient?
```

Comment:
You can't verify that just based on the bytes of the Orchard section of the transaction. You need to have the data that was encrypted and encoded to produce those bytes. That is available in the PCZT. I think @str4d will need to explain this.